package.json

```diff
@@ -8,6 +8,7 @@
     "@npmcli/name-from-folder": "^1.0.1",
     "@npmcli/run-script": "^1.3.1",
     "bin-links": "^2.1.2",
+    "cacache": "^15.0.3",
```

Comment:
question: why cacache is only getting added now?

Reply:
Until now, Arborist didn't use cacache directly!  But cracking open tarballs requires a temp directory to extract into, and it just makes sense to use the temp directory that cacache provides, since the rest of npm does.